src/DeviceStatusPage/DeviceInfoCard.tsx

```diff
@@ -83,6 +110,10 @@ const DeviceInfoCard = ({
       <p>
         <ConnectionStatus status={device.connectionStatus} />
       </p>
+      <h6>Deletion in progress</h6>
+      <p>
+        <DeletionStatus status={device.deletionInProgress} />
```

Comment:
When `deletionInProgress` is `true`, let's disable action buttons since they are not useful anymore.
For sure let's disable the button to delete the device.